### PR TITLE
Don't use workaround on latest version of Vim

### PR DESCRIPTION
**Pull request by mattn**
:clap: 

https://github.com/vim/vim/pull/3823

terminal6

### CODE REVIEW

File: autoload/async/job.vim [hunk 1]
```diff
@@ -184,8 +184,12 @@ function! s:job_send(jobid, data) abort
     if l:jobinfo.type == s:job_type_nvimjob
         call jobsend(a:jobid, a:data)
     elseif l:jobinfo.type == s:job_type_vimjob
-        let l:jobinfo.buffer .= a:data
-        call s:flush_vim_sendraw(a:jobid, v:null)
+        if has('patch-8.1.0818')
+            call ch_sendraw(l:jobinfo.channel, a:data)
```
Comment (prabirshrestha) [inline autoload/async/job.vim:188]: does vim handle overflows?

Comment (prabirshrestha) [reply autoload/async/job.vim:188]: seems like vim handles it.

**Comment by prabirshrestha** [comment]
Merged. Thanks

**Comment by mattn** [comment]
Now, the code completion got performance that is same as neovim. For the next, I'm trying to improve performance of signs. :)

**Pull request merged**